thunder/core/prims.py

```diff
@@ -261,6 +261,8 @@ class PrimIDs(Enum):
     # Memory access methods
     ITEM = auto()
     COPY_ = auto()
+    #
```

Comment:
@jjsjann123 Could we have a brief in-file comment here?

Reply:
oops, let me do that.